ng-ncc-app/src/app/components/account-balance/account-balance.component.spec.ts

```diff
@@ -1,25 +1,27 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```

Comment:
`CUSTOM_ELEMENTS_SCHEMA` is used in this test to ignore component dependencies (i.e. having to add components that appear in this component to the TestBed).